RELEASE.md

```diff
@@ -9,6 +9,7 @@ Please follow the established format:
 
 ## Major features and improvements
 
+- Introduce `--include-hooks` option and deprecate `--ignore-plugins` from cli commands. (#1818)
```

Comment:
It says `--ignore-plugins` is deprecated, but from looking at the code it looks like it's removed?

Reply:
And is this going to be included in a breaking release?

Reply:
Deprecating and then removing would be the "right" process, but if no-one is using this flag I'm fine with removing straight away in a non-breaking release. Just make sure it's clear in the release notes 🙂